stats/stats.go

```diff
@@ -393,21 +390,56 @@ func (h *Histogram) Clone() *Histogram {
 // CopyFrom sets the content of this object to a copy of the src.
 func (h *Histogram) CopyFrom(src *Histogram) {
 	h.Counter = src.Counter
-	// we don't copy offset/divider as this assumes compatible src/dest
-	for i := 0; i < len(h.Hdata); i++ {
-		h.Hdata[i] += src.Hdata[i]
+	h.CopyHDataFrom(src)
+}
+
+func (h *Histogram) CopyHDataFrom(src * Histogram)  {
```

Comment:
can you add a comment for the method, including that in your current implementation the destination histogram "wins" 
(maybe we can add later some other method that tries to calculate the best/most compatible offset/divider pair for the 2 or N input histograms)

Reply:
Okay I added